endpointmanager/pkg/chplendpointquerier/meditechwebscraper.go

```diff
@@ -0,0 +1,70 @@
+package chplendpointquerier
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"strings"
+
+	"github.com/PuerkitoBio/goquery"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/chromedp/chromedp"
+)
+
+func Meditechwebscraper(CHPLURL string, fileToWriteTo string) {
```

Comment:
Kind of a nitpicky comment, but probably want to change CHPLURL variable to be relevant to meditech, since it does not refer to a CHPL url